pg_search/src/postgres/build_parallel.rs

```diff
@@ -318,16 +322,16 @@ impl WorkerBuildState {
             max_docs_per_segment,
         };
         let writer = SerialIndexWriter::open(indexrel, config, worker_number)?;
-        let schema = SearchIndexSchema::open(indexrel.oid())?;
+        let schema = SearchIndexSchema::open(indexrel)?;
         let categorized_fields = categorize_fields(indexrel, &schema);
         let key_field_name = schema.key_field().field_name();
         Ok(Self {
             writer: Some(writer),
             categorized_fields,
             key_field_name,
             per_row_context: PgMemoryContexts::new("pg_search ambuild context"),
-            indexrel: indexrel.clone(),
-            heaprel: indexrel.heap_relation().unwrap().clone(),
+            indexrel: Clone::clone(indexrel),
+            heaprel: Clone::clone(heaprel),
```

Comment:
Why `Clone::clone` rather than a call to clone? I don't see any method name conflicts.

Reply:
Good catch.  My first run at this was using `Arc<PgRelation>` and it made sense in that context.  I'll clean this up.